demos/http3/Makefile

```diff
@@ -4,17 +4,20 @@
 #
 #    LD_LIBRARY_PATH=../.. ./ossl-nghttp3-demo www.example.com:443
 
-CFLAGS  += -I../../include -g -Wall -Wsign-compare
+CFLAGS  += -I../../include -g -Wall -Wsign-compare --extra-warnings
```

Comment:
Please drop this change, I do not think `--extra-warnings`  is a real option of gcc or clang.

Reply:
Oops sorry my bad.